java/pom.xml

```diff
@@ -38,7 +38,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <protoc.version>3.19.4</protoc.version>
+        <protoc.version>3.21.9</protoc.version>
```

Comment:
We should keep this in sync with the version below (and there've been security fixes made to this since we last updated).